creator-node/scripts/run-tests.sh

```diff
@@ -108,6 +108,7 @@ if [ -z "${isCIBuild}" ]; then
   docker exec -i $DB_CONTAINER /bin/sh -c "psql -U postgres -tc \"SELECT 1 FROM pg_database WHERE datname = 'audius_creator_node_test'\" | grep -q 1 || psql -U postgres -c \"CREATE DATABASE audius_creator_node_test\""
 fi
 
+rm -rf $storagePath
```

Comment:
this is necessary because `set -e` causes the test to exit and subsequent runs could have local artifacts

Reply:
this technically is the "beginning" of the script. before this it's just functions to do checks and exporting env vars etc. i will add a comment about this